source/index.txt

```diff
@@ -32,145 +38,329 @@ Atlas App Services
-App Services to write and host a full application in a managed cloud
-environment backed by MongoDB Atlas.
+.. introduction::
+   
+   Atlas Application Services are fully-managed backend services and
+   APIs that help you build apps and services with MongoDB Atlas faster.
+   You can use the services to build apps, integrate Atlas to other
+   services, and connect to your Atlas data without managing servers.
+   
+   Get started with your first app on our :doc:`free tier </billing>`.
+   
+   .. button:: Start Building
+      :uri: https://www.mongodb.com/realm/register?tck=docs_CTA_app_services
 
-.. button:: Create an Account
-   :uri: https://www.mongodb.com/realm/register?tck=docs_CTA_app_services
+.. image:: /images/plp/apis-tech.png
+   :alt: Atlas App Services Illustration
 
+.. kicker:: What You Can Do
 
-.. card-group::
-   :style: extra-compact
-   :columns: 2
+Build Data-Driven Apps And Services
+-----------------------------------
 
-   .. card::
-      :headline: Create a New App
-      :url: https://www.mongodb.com/docs/atlas/app-services/manage-apps/create/create-with-ui/
-      :icon: /images/icons/realm/cross-platform.svg
-      :icon-alt: App Icon
-      
-      {+apps+} contain all of your hosted services. Learn how to set up your
-      Atlas cluster and create a new serverless application.
+Services like Device Sync, APIs, and Triggers make it easy to build and
+run virtually any app or backend service on top of Atlas.
 
-   .. card::
-      :headline: Build a Serverless App Backend
-      :url: https://www.mongodb.com/docs/atlas/app-services/tutorial/backend/
-      :icon: /images/icons/realm/realm-lightning.svg
-      :icon-alt: DB Triggers Icon
-      
-      See how to use HTTPS endpoints and triggers to run a serverless GitHub issue
-      tracker.
+.. tabs::
 
-   .. card::
-      :headline: Explore Template Applications
-      :url: https://www.mongodb.com/docs/atlas/app-services/template-apps/
-      :icon: /images/icons/realm/realm-mobile.svg
-      :icon-alt: App Services Mobile Icon
-      
-      Explore working applications to learn how to implement Device Sync in
-      mobile applications or use App Services.
+   .. tab:: Sync Data Across Devices
+      :tabid: device-sync
 
-   .. card::
-      :headline: Learn More About Services
-      :url: https://www.mongodb.com/docs/atlas/app-services/introduction/
-      :icon: /images/icons/realm/graphql.svg
-      :icon-alt: GraphQL Icon
+      .. procedure::
+
+         .. step:: Install a Realm SDK
+
+            Device Sync uses Realm's native language objects to define a
+            data model and work with data.
+            
+            To install an SDK, see the :realm:`Realm SDK documentation
+            </>` for your language and platform.
+
+         .. step:: Enable Sync
+
+            In your Atlas App, specify which fields users can query and
+            define read and write permissions in Dev Mode. Once Sync is
```

Comment:
Nit: this reads to me like Dev Mode is what enables you to "specify which fields users can query and define read and write permissions." I'm not sure we need to mention Dev Mode at all here, if this is a high-level overview, so I'm wondering if we can just stop at "define read and write permissions."

Reply:
Tweaked the wording here, removed dev mode specifically